### PR TITLE
nav edits

### PR DESCRIPTION
**Pull request by mbouzi**
- fix tardigrade nav to look more like it does here https://www.figma.com/file/g7doUVwCahjA5YHJNQnNd7A5/Tardigrade.io?node-id=4493%3A314

### CODE REVIEW

**Comment by mbouzi** [comment]
@boshevski was not able to recreate the overlapping content issue, what browser are you using? Everything else had been addressed!

File: layouts/partials/legalNav.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
-<nav class="navbar navbar-expand-lg">
+<nav class="navbar navbar-expand-lg p-0">
 	<div class="container">
-    <a class="navbar-brand" href="/">
-        <img alt="Tardigrade lobo" src="/img/tar-logo-blue-horizontal.svg" style="margin-top:-14px;"/>
-    </a>
+		<a class="navbar-brand" href="/">
+				<img alt="Tardigrade lobo" src="/img/tar-logo-blue-horizontal.svg" style="margin-top:-14px;"/>
```
Comment (boshevski) [inline layouts/partials/legalNav.html]: Typo: Tardigrade lobo :))

File: layouts/partials/legalNav.html [hunk 2]
```diff
@@ -16,11 +16,11 @@
 	              <span class="sr-only">(current)</span>
 	            </button>
 	            <div class="dropdown-menu">
-	              <p class="dropdown-item" href="#">For Developers</p>
+	              <p class="dropdown-header dropdown-item" href="#">For Developers</p>
```
Comment (boshevski) [inline layouts/partials/legalNav.html]: Remove class "dropdown-item"

File: layouts/partials/legalNav.html [hunk 3]
```diff
@@ -29,10 +29,10 @@
 	              <span class="sr-only">(current)</span>
 	            </button>
 	            <div class="dropdown-menu">
-	              <p class="dropdown-item" href="#">Our Partners</p>
+	              <p class="dropdown-header dropdown-item" href="#">Our Partners</p>
```
Comment (boshevski) [inline layouts/partials/legalNav.html]: Remove class "dropdown-item"

File: layouts/partials/nav.html [hunk 4]
```diff
@@ -17,11 +17,11 @@
 	              <span class="sr-only">(current)</span>
 	            </button>
 	            <div class="dropdown-menu">
-	              <p class="dropdown-item" href="#">For Developers</p>
+	              <p class="dropdown-header dropdown-item" href="#">For Developers</p>
```
Comment (boshevski) [inline layouts/partials/nav.html]: Remove class "dropdown-item"

File: layouts/partials/nav.html [hunk 5]
```diff
@@ -30,10 +30,10 @@
 	              <span class="sr-only">(current)</span>
 	            </button>
 	            <div class="dropdown-menu">
-	              <p class="dropdown-item" href="#">Our Partners</p>
+	              <p class="dropdown-header dropdown-item" href="#">Our Partners</p>
```
Comment (boshevski) [inline layouts/partials/nav.html]: Remove class "dropdown-item"

File: assets/scss/_home.scss [hunk 6]
```diff
@@ -127,22 +135,26 @@
...
+		}
   }
   img {
     float: right;
     position: relative;
     top: 7px;
     display: none
   }
-  p {
+  .dropdown-header {
```
Comment (boshevski) [inline assets/scss/_home.scss:152]: Add **margin-top: 8px;** to **.dropdown-header**

File: assets/scss/_home.scss [hunk 7]
```diff
@@ -151,6 +163,23 @@
   }
 }
 
+.dropdown-menu.show {
+	@media only screen and (max-width: 991px) {
+		display: inline;
+		.dropdown-item {
+			color: #fff;
+		}
+		.dropdown-header {
```
Comment (boshevski) [inline assets/scss/_home.scss:213]: Add **margin-top: 0;** to **.dropdown-header**

**Comment by mbouzi** [comment]
@boshevski did my best to address the overlapping. Let me know if you have a better solution, or any ideas on what to do for the partner page.

**Comment by mbouzi** [comment]
@boshevski looks good!

**Pull request merged**